test/prism/heredoc_dedent_test.rb

```diff
@@ -7,8 +7,13 @@ class HeredocDedentTest < TestCase
     filepath = File.expand_path("fixtures/tilde_heredocs.txt", __dir__)
 
     File.read(filepath).split(/(?=\n)\n(?=<)/).each_with_index do |heredoc, index|
+      # The first example in this file has incorrect dedent calculated by
+      # TruffleRuby so we skip it.
+      next if index == 0 && RUBY_ENGINE == "truffleruby"
```

Comment:
I don't know how likely this is, but this means we'll fail on truffleruby if we move around the examples in this file. Should we mark that in the fixtures file itself too?

Reply:
I think it's alright for now - this test will fail CI so we will be able to catch it.